frontend/src/v5/ui/controls/imageWithExtraCount/imageWithExtraCount.component.tsx

```diff
@@ -15,21 +15,21 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Image, ExtraImages, OverlappingContainer } from './commentImage.styles';
+import { Image, ExtraImages, OverlappingContainer } from './imageWithExtraCount.styles';
```

Comment:
launching a second time the carousel fails.
![image carousel error](https://github.com/user-attachments/assets/96b939af-f9de-4c27-9fc8-756c2c7a46f4)